src/components/page-sections/our-partners/OurPartners.jsx

```diff
@@ -35,10 +37,6 @@ const OurPartners = () => (
             <Image
               src={value}
               width="100%"
-              style={{
-                filter: "grayscale(1)",
-                opacity: index === 1 || index === 3 ? "0.3" : 1,
-              }}
```

Comment:
If you want to keep those lines (which is a good idea), just put those lines in comments.
Do not move those lines at the end of the files (see other comment).

Reply:
committed this change: https://github.com/HackYourFutureBelgium/hackyourfuture.be/commit/580ff590904f57a6f753a8312b550909e48dfdab

I'd moved it to the bottom because I remembered learning once that comments don't work in JSX.  I could have double checked and tested it myself

Reply:
this wasn't hard to find: https://wesbos.com/react-jsx-comments